catalog/spatial/geocoding/spatial-geocoding-api/src/test/java/org/codice/ddf/spatial/geocoding/TestBase.java

```diff
@@ -20,12 +20,13 @@
 
     protected void verifyGeoEntry(final GeoEntry geoEntry, final String name, final double latitude,
             final double longitude, final String featureCode, final long population,
-            final String alternateNames) {
+            final String alternateNames, final String countryCode) {
         assertEquals(name, geoEntry.getName());
         assertEquals(latitude, geoEntry.getLatitude(), 0);
         assertEquals(longitude, geoEntry.getLongitude(), 0);
         assertEquals(featureCode, geoEntry.getFeatureCode());
         assertEquals(population, geoEntry.getPopulation());
         assertEquals(alternateNames, geoEntry.getAlternateNames());
+        assertEquals(countryCode, geoEntry.getCountryCode());
```

Comment:
This is a little out of scope, but these really should be `assertThat` vs `assertEquals`